src/test/java/resources/batch.feature

```diff
@@ -94,6 +127,10 @@ Feature: Batch Details API test
   Scenario: BD-010 Batch summary with failure percent API Test
     Given I have a batch id from previous scenario
     And I have tenant as "gorilla"
+    And I have the demo csv file "ph-ee-bulk-demo-6.csv"
+    And I have clientCorrelationId as "9051df83-e13c-4d6c-a850-220874db737a"
```

Comment:
does this hardcoded id create unique key error?

Reply:
no